### PR TITLE
Added message when large sponsor logo is uploaded

### PR DESCRIPTION
**Pull request by clloyd**
When an image is over 500px in any dimension a message is added next to the image information

### CODE REVIEW

**Comment by clloyd** [comment]
@steppenwells

File: public/components/SponsorshipEdit/SponsorLogo.react.js [hunk 1]
```diff
@@ -52,6 +52,19 @@ export default class SponsorEdit extends React.Component {
...
 
+
+  renderImageError(imageAsset) {
+    if (imageAsset.height < 500 && imageAsset.width < 500 ) {
+      return false;
+    }
+
+    return (
+      <div className="tag-edit__image__error">
+        <i className="i-info-grey" /> The uploaded logo is larger than required.
```
Comment (steppenwells) [inline public/components/SponsorshipEdit/SponsorLogo.react.js]: not convinced about this wording, this is a warning so "required" is misleading

File: public/components/SponsorshipEdit/SponsorLogo.react.js [hunk 2]
```diff
@@ -52,6 +52,19 @@ export default class SponsorEdit extends React.Component {
...
 
+
+  renderImageError(imageAsset) {
+    if (imageAsset.height < 500 && imageAsset.width < 500 ) {
+      return false;
+    }
+
+    return (
+      <div className="tag-edit__image__error">
+        <i className="i-info-grey" /> Note: The uploaded logo is greater than 500px.
```
Comment (clloyd) [inline public/components/SponsorshipEdit/SponsorLogo.react.js]: @steppenwells It now just highlights the facts, which is probably more useful

**Comment by steppenwells** [comment]
Message is now better. 👍

**Pull request merged**